index.html

```diff
@@ -284,6 +284,9 @@ <h2>Scope</h2>
 document as input, the WG will produce a NOTE at the end of the process that is
 a refined Use Cases document.
           </li>
+          <li>
+Recommend requirements for DID methods, including which operations must be supported, which must be described, and which are optional.
```

Comment:
I'm not sure I understand that text. 

Maybe this would be better?
```
Recommend a list of operations, with associated descriptions, which are expected 
to be included in future DID method specifications, including which operations 
are expected to be required in all DID method implementations based on those 
specifications, and which operations are expected to be optional.
```

Reply:
That's a lot of text to say "some operations are required, some are optional" :) -- attempted to simplify in 2aa2cedba17dba1377d7b36d687257d678851800. Does that work for you @TallTed and @jandrieu?